tasks/prime.py

```diff
@@ -242,8 +244,10 @@ def on_start(self):
         email = json_resp["email"]
         user_id = json_resp["id"]
 
-        origin_duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=29"))
-        current_station_id = origin_duty_stations.json()[0]["id"]
+        # Setup customer profile
+        duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=palms"))
+        stations = duty_stations.json()
+        current_station_id = stations[0]["id"]
 
         overrides = {
```

Comment:
I wonder if it's worth adding a custom personal_email or first_name/last_name values so that we can easily locate these load test users in the database.  I know it won't impact what shows up on the login screen since that is set during user creation and is only editable through the admin.

Reply:
I think this is worth thinking about. Is there a specific issue or use case that we are trying to solve? When running locally I don't this matters all that much as their are other alternatives that the user can decide which method works for them (eg. printing the email, providing an email in the overrides). When doing this in a deployed environment, I think it depends on why someone may want this information. If multiple users are created in a single test specifying a single email may not actually help narrow down the selection. 

I'd lean towards not adding this functionality unless there is a specific need/issue that we are currently trying to solve. If there is a common issue we can default to a specific methodology.  I think this functionality would make sense a future state when this functionality is explicitly needed.

Reply:
Ie. When would one want to look at the customer from the DB? The load tests that we are running don't necessarily modify the move that we create in this setup flow.

Reply:
That's true it won't be very useful if this becomes the basis for customer load tests because there will be thousands or so.  Right now it's just a single move in the system so yea I think it might be nice to have a stable identifier of some sort. I also wanted to have a way to distinguish potential data already in the system from load test created moves.  When a request fails it does log the payload when running locally so you can work backwards from the ids.